src/Cake.Common.Tests/Unit/Tools/DotNet/MSBuild/DotNetMSBuildBuilderTests.cs

```diff
@@ -801,6 +802,20 @@ public void Should_Add_Host_Arguments()
                 // Then
                 Assert.Equal("--diagnostics msbuild", result.Args);
             }
+
+            [Fact]
+            public void Should_Use_Node_Reuse_If_Specified()
+            {
+                // Given
+                var fixture = new DotNetMSBuildBuilderFixture();
+                fixture.Settings.NodeReuse = true;
+
+                // When
+                var result = fixture.Run();
+
+                // Then
+                Assert.Equal("msbuild /nr:true", result.Args);
```

Comment:
```suggestion
                Assert.Equal("msbuild /nodeReuse:true", result.Args);
```

Let's use the longer name version to make it easier to understand